### PR TITLE
Updated instructions and code for DALL-E 3

### PR DESCRIPTION
**Pull request by GraemeMalcolm**
Updates so that the learner uses a DALL-E 3 model in East US or Sweden.
Requires corresponding updates to Learn and ILT content.

### CODE REVIEW

File: Instructions/Exercises/05-generate-images.md [hunk 1]
```diff
@@ -7,30 +7,32 @@ lab:
...
 
 3. Wait for deployment to complete. Then go to the deployed Azure OpenAI resource in the Azure portal.
 
 ## Explore image-generation in the DALL-E playground
 
 You can use the DALL-E playground in **Azure OpenAI Studio** to experiment with image-generation.
 
-1. In the Azure portal, on the **Overview** page for your Azure OpenAI resource, use the **Explore** button to open Azure OpenAI Studio in a new browser tab. Alternatively, navigate to [Azure OpenAI Studio](https://oai.azure.com) directly.
-2. In the **Playground** section, select the **DALL-E** playground.
+1. In the Azure portal, on the **Overview** page for your Azure OpenAI resource, use the **Explore** button to open Azure OpenAI Studio in a new browser tab. Alternatively, navigate to [Azure OpenAI Studio](https://oai.azure.com) directly at `https://oai.azure.com`.
```
Comment (LayZeeDK) [inline Instructions/Exercises/05-generate-images.md:34]: suggestion: It seems redundant to have the same link twice in the same phrase. Also, this is not a Markdown link.

**Review by LayZeeDK — CHANGES_REQUESTED** [review]
Suggest change.

File: Instructions/Exercises/05-generate-images.md (hunk 1 above)
Comment (GraemeMalcolm) [reply Instructions/Exercises/05-generate-images.md:34]: The additional code-formatted link is used in our hosted lab environment to make it easier to auto-type the link into a hosted VM.

Comment (LayZeeDK) [reply Instructions/Exercises/05-generate-images.md:34]: Ah, I see, it's the *Click to type into the lab VM* feature 👍

**Pull request merged**